internal/backends/linuxkms/Cargo.toml

```diff
@@ -20,6 +20,7 @@ renderer-skia = ["renderer-skia-opengl"]
 renderer-skia-vulkan = ["i-slint-renderer-skia/vulkan", "vulkano"]
 renderer-skia-opengl = ["i-slint-renderer-skia/opengl", "drm", "gbm", "gbm-sys", "glutin", "raw-window-handle"]
 renderer-femtovg = ["i-slint-renderer-femtovg", "drm", "gbm", "gbm-sys", "glutin", "raw-window-handle"]
+renderer-software = ["i-slint-core/software-renderer-systemfonts", "i-slint-core/software-renderer-rotation", "dep:bytemuck"]
```

Comment:
Ooops, good point about it becoming public API. That's not intentional in this PR.

It's a bummer, because the feature itself works well (used in mcu as well as here). But we're not sure yet about the exact API, i.e. explicit in SoftwareRenderer as property, as a parameter to render(), or implicit in the Window, right?